js/components/time.js

```diff
@@ -40,19 +42,31 @@ Fliplet.FormBuilder.field('time', {
   },
   methods: {
     initTimePicker: function() {
+      if (this.timePicker || !this.$refs.timePicker) {
+        return;
+      }
+
       var $vm = this;
 
-      this.timepicker = $($vm.$refs.timepicker).timeEntry()
-        .on('change', function(event) {
-          $vm.value = event.target.value;
-        });
+      this.timePicker = Fliplet.UI.TimePicker(this.$refs.timePicker, {
+        required: this.required || this.autofill === 'always',
+        value: this.value
+      });
 
-      this.timepicker.timeEntry('setTime', $vm.value);
+      this.timePicker.change(function(value) {
+        $vm.value = value;
+        $vm.updateValue();
+      });
     }
   },
   computed: {
     isApplyCurrentDateField: function() {
       return this.autofill === 'always' || this.autofill === 'default';
+    },
+    readonlyValue: function() {
```

Comment:
Should this use camel case as `readOnlyValue`?

Reply:
@squallstar Good question. I stuck with `readonly` because that's how we've used it throughout the form component.

![image](https://user-images.githubusercontent.com/290733/169577704-c3929f57-ca67-47d4-ba27-28cad7ad56aa.png)